### PR TITLE
Feature wake cleanup

### PR DESCRIPTION
**Pull request by rachealerhard**
Mostly a code cleanup and minor bug fix to plotting, plus the following changes:
- Switches the propeller_single_point analysis to be for a single prop rather than network with a specified prop number
- Updates the rotor tip loss function with a tolerance to prevent nans of tip-station airfoils

### CODE REVIEW

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py [hunk 1]
```diff
@@ -140,19 +140,11 @@ def compute_inflow_and_tip_loss(r,R,Wa,Wt,B):
...
-    f                 = (B/2.)*(1.-r/R)/lamdaw
-    f[f<0.]           = 0.
-    
-    piece             = np.exp(-f)
-    F                 = 2.*np.arccos(piece)/np.pi
 
-    Rtip = R
     et1, et2, et3, maxat = 1,1,1,-np.inf
-    tipfactor = B/2.0*(  (Rtip/r)**et1 - 1  )**et2/lamdaw**et3
+    tipfactor = B/2.0*(  (R/r)**et1 - 1  )**et2/lamdaw**et3 + 1e-3 # tolerance to prevent nan
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: I ran into this issue before. [Check this out](https://github.com/suavecode/SUAVE/blob/jax-nexus_grad/trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py#L143).

File: trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py [hunk 2]
```diff
@@ -129,18 +130,18 @@ def mini_mission(altitude, speed):
                 D = -results.segments.single_point.conditions.frames.wind.drag_force_vector[0][0]
 
                 # Determine Propeller Power at Altitude and Speed
-
-                P = propeller_single_point(vehicle.networks.battery_propeller,
-                                           analyses,
+                prop_key = list(vehicle.networks.battery_propeller.propellers.keys())[0]
```
Comment (planes) [inline trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py:133]: Wait, this won't work as intended anymore. This was supposed to run a vehicle not just a propeller. The V-h diagrams are nonsense if it's just a single prop.

File: trunk/SUAVE/Methods/Propulsion/Rotor_Wake/Fidelity_One/generate_fidelity_one_wake_shape.py [hunk 3]
```diff
@@ -144,9 +143,12 @@ def generate_fidelity_one_wake_shape(wake,rotor):
     
 
     # extract airfoil trailing edge coordinates for initial location of vortex wake
-    a_sec        = rotor.airfoil_geometry   
+    if rotor.airfoil_data == None:
```
Comment (planes) [inline trunk/SUAVE/Methods/Propulsion/Rotor_Wake/Fidelity_One/generate_fidelity_one_wake_shape.py]: Do this in the analysis instead for the method

File: trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py (hunk 2 above)
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py:133]: propeller_single_point() only used the provided or first indexed propeller in the list of propellers anyways. No functionality has changed with this PR, if this is supposed to do something different than it wasn't doing it before.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py (hunk 1 above)
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: Modified

File: trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py (hunk 2 above)
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py:133]: propeller_single_point() only used the provided or first indexed propeller in the network previously. No functionality has changed with this PR, so if this is supposed to do something different then it wasn't doing it before. We can address this issue in a new PR.

File: trunk/SUAVE/Methods/Propulsion/Rotor_Wake/Fidelity_One/generate_fidelity_one_wake_shape.py (hunk 3 above)
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Propulsion/Rotor_Wake/Fidelity_One/generate_fidelity_one_wake_shape.py]: Updated

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py [hunk 4]
```diff
@@ -142,8 +142,9 @@ def compute_inflow_and_tip_loss(r,R,Wa,Wt,B):
     lamdaw[lamdaw<0.] = 0.
 
     et1, et2, et3, maxat = 1,1,1,-np.inf
-    tipfactor = B/2.0*(  (R/r)**et1 - 1  )**et2/lamdaw**et3 + 1e-3 # tolerance to prevent nan
-    tipfactor[tipfactor<0.]   = 0.
+    tipfactor = B/2.0*(  (R/r)**et1 - 1  )**et2/lamdaw**et3 
+    tipfactor[np.where(tipfactor==0)] = 1e-6 # tolerance to prevent nan
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: this is not the line that needs to be fixed.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py [hunk 5]
```diff
@@ -142,8 +142,9 @@ def compute_inflow_and_tip_loss(r,R,Wa,Wt,B):
     lamdaw[lamdaw<0.] = 0.
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: this is the line that needs to be fixed:
lamdaw[lamdaw<0.] = 1e-12

File: trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py (hunk 2 above)
Comment (planes) [reply trunk/SUAVE/Methods/Performance/electric_V_h_diagram.py:133]: Oh jeez, we need to make an issue then.

File: trunk/SUAVE/Components/Energy/Converters/Rotor.py [hunk 6]
```diff
@@ -583,12 +586,12 @@ def vec_to_vel(self):
...
-        """This rotates from the systems body frame to the propellers velocity frame
+        """This rotates from the system's body frame to the propeller's velocity frame
 
         Assumptions:
-        There are two propeller frames, the vehicle frame describing the location and the propeller velocity frame
-        velocity frame is X out the nose, Z towards the ground, and Y out the right wing
-        vehicle frame is X towards the tail, Z towards the ceiling, and Y out the right wing
+        There are two propeller frames, the vehicle frame describing the location and the propeller velocity frame.
+        Velocity frame is X out the nose, Z towards the ground, and Y out the right wing
+        Vehicle frame is X towards the tail, Z towards the ceiling, and Y out the right wing
```
Comment (WallyMaier) [inline trunk/SUAVE/Components/Energy/Converters/Rotor.py:594]: Punctuation here pls

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py (hunk 5 above)
Comment (rachealerhard) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: updated

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py [hunk 7]
```diff
@@ -138,21 +138,13 @@ def compute_inflow_and_tip_loss(r,R,Wa,Wt,B):
...
-    f[f<0.]           = 0.
-    
-    piece             = np.exp(-f)
-    F                 = 2.*np.arccos(piece)/np.pi
-
-    Rtip = R
+    lamdaw             = r*Wa/(R*Wt)
+    lamdaw[lamdaw<=0.] = 1e-12
+
     et1, et2, et3, maxat = 1,1,1,-np.inf
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: these are effectively not used

Comment (rachealerhard) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: Yep; they are tunable parameters that change the tip loss function. Can be adjusted, and may be updated in the future.

Comment (planes) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: maxat is not used. Default the rest as inputs above.

Comment (rachealerhard) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/BET_calculations.py]: done

**Review by planes — APPROVED** [review]

**Pull request merged**